core/snippets/snippets.py

```diff
@@ -29,6 +36,12 @@
     context_map[lang] = ctx
 
 
+def get_setting_dir():
+    if not setting_dir.get():
+        return None
+    return Path(actions.path.talon_user()) / setting_dir.get()
```

Comment:
I think the idea was to do what we do with Cursorless, where we only prepend talon user if it's not an absolute path, right?

Reply:
done